index.html

```diff
@@ -1419,10 +1423,29 @@
           return the <a>RemotePlayback</a> instance associated with the
           <a>media element</a>.
         </p>
-        <p>
-          The <dfn data-dfn-for='HTMLMediaElement'>disableRemotePlayback</dfn>
-          IDL attribute MUST reflect the content attribute of the same name.
-        </p>
+        <section>
+          <h4>
+            The <code><dfn data-dfn-for='HTMLMediaElement'>disableRemotePlayback</dfn></code> attribute
+          </h4>
+          <p>
+            Some pages may wish to disable remote playback of a media element;
+            for example, they may prefer to use a <a>PresentationRequest</a> to
+            present a complete document on a presentation screen.  To support
+            this use case, a new <code>disableRemotePlayback</code> is added to
+            the list of content attributes for <code>audio</code>
+            and <code>video</code> elements.
+          </p>
+          <p>
+            A corresponding
+            <a data-link-for='HTMLMediaElement'>disableRemotePlayback</a>
+            IDL attribute which reflects the value each element’s
```

Comment:
I see that sentence matches the one in the Subresource Integrity spec, but I cannot parse "the value each element's disableRemotePlayback". Is that proper English? Perhaps "the value of the element's disableRemotePlayback content attribute"?

Reply:
Looks like a typo in Subresource Integrity.  Fixed here and submitted https://github.com/w3c/webappsec-subresource-integrity/pull/74